backend/src/main/kotlin/fr/gouv/cnsp/monitorfish/domain/use_cases/mission/mission_actions/ProcessMissionAction.kt

```diff
@@ -0,0 +1,30 @@
+package fr.gouv.cnsp.monitorfish.domain.use_cases.mission.mission_actions
+
+import fr.gouv.cnsp.monitorfish.config.UseCase
+import fr.gouv.cnsp.monitorfish.domain.entities.mission.mission_actions.MissionAction
+import fr.gouv.cnsp.monitorfish.domain.exceptions.CodeNotFoundException
+import fr.gouv.cnsp.monitorfish.domain.repositories.PortRepository
+import org.slf4j.LoggerFactory
+
+@UseCase
+class ProcessMissionAction(
```

Comment:
```suggestion
class EnrichMissionAction(
```
Je dirais `Enrich` pour expliciter l'ajout de proriété, wdyt ?